documentation.lisp

```diff
@@ -1598,6 +1605,30 @@ yourself.
 
 See CLIENT
 See ACCOUNT
+See STATUS")
+
+  (function bookmarks
+    "Returns a list of statuses bookmarked by the user.
+
+See CLIENT
+See STATUS")
+
+  (function bookmarks
+    "Returns a list of bookmarked STATUS.
+
+See CLIENT
+See STATUS")
```

Comment:
You can only have one docstring per function binding, so you have to merge the two somehow.

Reply:
Oh, sorry, for some reason i duplicated the documentation of the `bookmark` function.